.github/actions/run-tests/action.yml

```diff
@@ -0,0 +1,30 @@
+name: run-tests
+description: 'Runs tests according to input parameters'
+
+inputs:
+  tools-version:
+    description: Which toolchain to build
+    required: false
+    default: 'riscv-tools'
+  group-key:
```

Comment:
This isn't needed anymore since the cache retrieval is done outside of this composite action.

Reply:
ok